client/src/components/LoginSession/LoginSession.js

```diff
@@ -25,10 +26,12 @@ function LoginSession(props) {
         props.logout();
     }
 
-    const created = moment(props.Created);
-    const createdElapsed = moment.utc(props.Created).fromNow();
-    const lastAccessed = moment(props.LastAccessed);
-    const lastAccessedElapsed = moment.utc(props.LastAccessed).fromNow();
+    moment.locale(ss.i18n.currentLocale);
+    const format = 'L LT';
+    const created = moment.utc(props.Created).local();
+    const createdElapsed = created.fromNow();
+    const lastAccessed = moment.utc(props.LastAccessed).local();
+    const lastAccessedElapsed = lastAccessed.fromNow();
```

Comment:
The logic for generating the tool tip is substantially more complicated with this PR. To make things a bit more simple to understand, I would suggest taking all the Tooltip/Time-interval logic and move it to its own functional component within the same file:

```jsx
function ToolTipInterval({ Created, LastAccessed, IsCurrent }) {
    moment.locale(i18n.detectLocale());
    const format = 'L LT';
    const created = moment.utc(Created).local();
    const createdElapsed = created.fromNow();
    const lastAccessed = moment.utc(LastAccessed).local();
    const lastAccessedElapsed = lastAccessed.fromNow()

    const lastActiveStr = IsCurrent ?
        i18n.inject(
            i18n._t('SessionManager.AUTHENTICATED', 'authenticated {createdElapsed}...'),
            { createdElapsed }
        )
        : i18n.inject(
            i18n._t('SessionManager.LAST_ACTIVE', 'last active {lastAccessedElapsed}...'),
            { lastAccessedElapsed }
        );
    const activityTooltip = i18n.inject(
        i18n._t('Admin.ACTIVITY_TOOLTIP_TEXT', 'Signed in {signedIn}, Last active {lastActive}'),
        {
            signedIn: created.format(format),
            lastActive: lastAccessed.format(format)
        }
    );

    return (
        <span data-toggle="tooltip" data-placement="top" title={activityTooltip}>
            , {lastActiveStr}
        </span>
    );
}
```

Then I would simply call this in the `<LoginSession />`:
```jsx
<ToolTipInterval Created={props.Created} LastAccessed={props.LastAccessed} IsCurrent={props.IsCurrent} />
```

Reply:
This refactor seems outside the scope of this issue.  Also I'm not really a fan of what's in a classic sense, 2 classes in 1 file.

Reply:
> 2 classes in 1 file

When you're using functional components it's common practice to split them into small manageable component ... it's not any different than having many methods on a single class. It's a relatively common react pattern.

And it's not a refactor either ... it's like if I asked you to take a big method and split it in two smaller method so it's easier to understand.

Reply:
If we're not going to move the Tooltip to a separate component, then we should at least move that logic around into more discernible code blocks and provide some comments to explain what is happening ... still think the separate component approach is superior.

Reply:
It kind of is a refactor, this PR has only added a single line to the code block `moment.locale(i18n.detectLocale());` and a pair of `.local()` to existing lines.  The code was already there, it was already fine and I had zero problems understanding it before, so I see little need to rewrite it.

Reply:
Sometimes you add a bit of code to the a existing function that put it beyond the point where it's sensible to keep it together. I think we've reached this point. 

You literally just need to copy-paste the example I gave you in the first comment. And for the time you spent arguing with me you could have done it already.